htsget.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
```

Comment:
add `set -xe` to script for verbose output and error messages

Reply:
Added the script as requested.